include/wx/gtk/mimetype.h

```diff
@@ -23,7 +23,11 @@
 class WXDLLIMPEXP_CORE wxGTKMimeTypesManagerImpl : public wxMimeTypesManagerImpl
 {
 protected:
+#if defined(__UNIX__)
     wxString GetIconFromMimeType(const wxString& mime) wxOVERRIDE;
+#else
+    wxString GetIconFromMimeType(const wxString& mime);
```

Comment:
../src/gtk/mimetype.cpp:22:77: error: no 'wxString wxGTKMimeTypesManagerImpl::GetIconFromMimeType(const wxString&)' member function declared in class 'wxGTKMimeTypesManagerImpl'
 wxString wxGTKMimeTypesManagerImpl::GetIconFromMimeType(const wxString& mime)
                                                                             ^
../src/gtk/mimetype.cpp:54:55: error: expected declaration before end of line
         wxGCC_WARNING_SUPPRESS(deprecated-declarations)